dts/bindings/display/ilitek,ili9xxx-common.yaml

```diff
@@ -7,23 +7,6 @@ description: ILI9XXX display controllers common properties.
 include: [spi-device.yaml, display-controller.yaml]
 
 properties:
-  reset-gpios:
-    type: phandle-array
-    description: RESET pin.
-
-      The RESET pin of ILI9340 is active low.
-      If connected directly the MCU pin should be configured
-      as active low.
-
-  cmd-data-gpios:
-    type: phandle-array
-    required: true
-    description: D/CX pin.
-
-      The D/CX pin of ILI9340 is active low (transmission command byte).
-      If connected directly the MCU pin should be configured
-      as active low.
```

Comment:
wait so the D/CX pin is not used anymore on these? but now it would be floating on these board, is that ok?

Reply:
It is still needed- but the D/CX pin is now handled by the MIPI DBI SPI driver. This is because not all MIPI DBI compatible IP will actually use GPIO to toggle this pin, so we must delegate the D/CX pin to the MIPI DBI driver layer

Reply:
Oh! sorry I saw `cd-gpios` and read `cs-gpios` for some reasons, ok makes sense.